crates/bevy_pbr/src/material.rs

```diff
@@ -715,6 +724,27 @@ pub fn extract_entities_needs_specialization<M>(
         // Update the entity's specialization tick with this run's tick
         entity_specialization_ticks.insert((*entity).into(), ticks.this_run());
     }
+    // Clean up any despawned entities
+    for entity in removed_view_visibility_components.read() {
```

Comment:
Does removing `ViewVisibility` counts as despawning? I guess from the point of view of the render world if an entity is not visible then it can be removed.

Reply:
Yeah this is probably inaccurate to say that's it's only a result of being despawned but the user removing the component is both something they shouldn't do in practice and will result in it not being rendered either way.